public/js/init.js

```diff
@@ -1,7 +1,7 @@
 // window.bootstrap = function() {
```

Comment:
is there a reason we have a couple of commented out code-blocks in source control?